localization/languages/pt-BR.json

```diff
@@ -31,7 +31,7 @@
         /* custom !bangs
         these are some of the items that show up when you press ! in the searchbar.
         Each one of these strings describes what the command will do when you run it. */
-        "showMoreBangs": null, //missing translation
```

Comment:
what did you mean by "showMoreBangs" ?

Reply:
Bangs are [shortcuts for searching on other websites](https://duckduckgo.com/bang), or for running commands in Min. This string is used for a button that shows more items in the list of bangs:

<img width="806" src="https://user-images.githubusercontent.com/10314059/73613869-6bed3800-45bf-11ea-9143-7cec47e3e4b6.png">